dungeon/src/contrib/utils/ICommand.java

```diff
@@ -1,15 +1,22 @@
 package contrib.utils;
 
 /**
- * ICommand is an interface that represents a command that can be executed and (optional) undone.
+ * ICommand is an interface that represents a command that can be executed and undone.
  *
  * @see contrib.entities.LeverFactory LeverFactory
  * @see contrib.systems.LeverSystem LeverSystem
  */
 public interface ICommand {
 
   /** Command that does nothing. */
-  ICommand EMPTY_COMMAND = () -> {};
+  ICommand NOOP =
```

Comment:
Huh. Wusste gar nicht, dass man in Interfaces auch Variablen definieren kann? Dachte immer, das wäre verboten?

An sich passt das so. Hätte allerdings eher sowas erwartet: `Supplier<Void> NOOP = () -> null;`. Das könnten dann Leute beim Implementieren als Funktion für `execute` oder `undo` nehmen. Aber in Deiner Schreibweise ist das gleich hübsch synchron: es sind beide Funktionen NOOP :) ...

Reply:
Hmm ja, wäre eigentlich noch was on top....
Quasi das NOOP_COMMAND (s.o)
und den Supplier<Void> NOOP
Dann kann ich selbst bauen oder das komplett empty ding nehmen

Reply:
jupp. weiss aber nicht, ob das nicht overkill ist. ich meine, eine leere funktion zu definieren ist jetzt nicht sooo aufwändig - und das im interface vorzuhalten zerklütert das deutlich. 

ich glaube, du solltest meinen kommentar einfach schließen und mergen :)